src/dredd.coffee

```diff
@@ -41,6 +42,24 @@ class Dredd
     @transactions = []
     @runner = new Runner(@configuration)
     configureReporters(@configuration, @stats, @tests, @runner)
+    @logProxySettings()
+
+  logProxySettings: ->
+    proxySettings = []
+    for envVariable in Object.keys(process.env)
```

Comment:
I'd consider `for for key, val of process.env` more readable and efficient, but YMMV.